sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodParameterUnused.CSharp8.cs

```diff
@@ -94,7 +94,7 @@ public void DoSomething(int a, int b)
     // https://github.com/SonarSource/sonar-dotnet/issues/3255
```

Comment:
Is this repro still valid since the FP appears to be wrong ? Should the corresponding issue be closed since the FP is not happening anymore ? If yes, do we know when this was fixed

Reply:
The issue was added recently and I think that we didn't see that the reproducer didn't worked due to the build which was always passing.

I've talked with @pavel-mikula-sonarsource on Friday (since he added the reproducer) and he will take a look and close the issue or ask for more details in the community thread to update it.

Reply:
Ok, thanks !